src/components/cursor-controller.js

```diff
@@ -69,7 +70,7 @@ AFRAME.registerComponent("cursor-controller", {
       }
 
       const interaction = AFRAME.scenes[0].systems.interaction;
-      let intersection;
+      this.intersection = null;
```

Comment:
Should this be moved up before the early out on line 69 (https://github.com/mozilla/hubs/pull/1730/files#diff-bafcf688d672a56b722fa967f5b16acbR69) ?